Cargo.toml

```diff
@@ -24,9 +24,10 @@ specs = "0.14.3"
 specs-hierarchy = "0.3.1"
 shrev = "1.1.1"
 nalgebra = "0.18.0"
-ncollide3d = "0.19.1"
```

Comment:
Why down to `0.19` from `0.19.1`?

Reply:
0.19 essentially means 0.19.*, so it's not "down" but tbh I included the change on accident after mucking about a bunch. A future PR where I'm doing the Amethyst example will include a heavy reckoning with making sure duplicate packages don't end up causing type mismatches, so I'll revert this change